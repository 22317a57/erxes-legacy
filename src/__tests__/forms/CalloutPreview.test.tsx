import { mount, shallow } from 'enzyme';
import CalloutPreview from 'modules/forms/components/step/preview/CalloutPreview';
import * as React from 'react';
import * as renderer from 'react-test-renderer';

describe('CalloutPreview component', () => {
  const defaultProps = {
    calloutTitle: 'string'
  };

  test('renders shallow successfully', () => {
    shallow(<CalloutPreview {...defaultProps} />);
  });

  test('renders mount with default props', () => {
    const control = mount(<CalloutPreview {...defaultProps} />);
    const props = control.props();

    expect(props).toMatchObject(defaultProps);
  });

  test('snapshot matches', () => {
    const rendered = renderer
      .create(<CalloutPreview {...defaultProps} />)
      .toJSON();

    expect(rendered).toMatchSnapshot();
  });
});
