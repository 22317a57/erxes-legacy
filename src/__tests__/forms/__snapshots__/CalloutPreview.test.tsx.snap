// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalloutPreview component snapshot matches 1`] = `
<div>
  asdf
</div>
`;
